src/aliceVision/sfm/pipeline/regionsIO.cpp

```diff
@@ -279,7 +278,7 @@ bool loadFeaturesPerView(feature::FeaturesPerView& featuresPerView,
       }
     }
   }
-  return !invalid;
+  return true;
```

Comment:
The boolean check `invalid` is not updated anywhere inside the caller, unless I'm missing something? @servantftechnicolor

Reply:
Yes, you're right, but the load function may return a nullptr if it can't find the file. Look at the other functions, they contain a check on nullptr.

Reply:
Then the issue is to be fixed by adding a check on the return result by the load function and updating the boolean check accordingly, right ? Since even reverting to the previous behavior will still has the issue.

Reply:
Moved to a separate PR https://github.com/alicevision/AliceVision/pull/1575